src/main/resources/jenkins/plugins/publish_over_ssh/BapSshTransfer/config.jelly

```diff
@@ -89,6 +89,10 @@
             <f:checkbox default="${defaults.transfer.useAgentForwarding}"/>
         </f:entry>
 
+        <f:entry title="${%useSftpForExec}" field="useSftpForExec">
```

Comment:
I think it is preferred to use an optionalProperty tag here instead of the checkbox.